react-native/react/menubar/index.render.js.flow

```diff
@@ -9,8 +9,14 @@ export default class Render extends Component {
 }
 
 export type FolderInfo = {
-  folderName: string,
-  isPublic: boolean,
-  isEmpty: boolean,
-  openFolder: () => void
+    type: 'folder',
+    folderName: string,
+    isPublic: boolean,
+    isEmpty: boolean,
+    openFolder: () => void
+} | {
+    type: 'entry',
+    prefix: string,
+    isPublic: boolean,
+    openFolder: (val: string) => void
```

Comment:
The list renders items. I can render 2 types of items, regular folders and folder entires (public / private are different. public its just a pure entry, private has  your name as a prefix. We can make FodlerInfo just one type and have the disjoint be in the array the FolderList renderer takes (we talked about this on slack and i thought we decided we could go either way) i don't care either way so should i just do that?